portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,92 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: auto;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+  background-color: #EEE7DC;
+}
+
+h1 {
+  font-family: helvetica;
+  color: #007372;
+  text-align: center;
+}
+
+h2{
+
+    font-family: geneva;
+    text-align: center;
+}
+
+p {
+  font-family: geneva;
+  font-size: 15px;
+  margin-left: 200px;
+  margin-right: 200px;
+  color: black;
+  text-align: center;
+}
+
+img {
+  display: block;
+  margin-top: 15px;
+  border-radius: 50%;
+  width: 25%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+button {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #007372;
+  border: none;
+  color: white;
+  padding: 10px 20px;
+  border-radius: 5%;
+  text-align: center;
+  font: geneva;
+  text-decoration: none;
+  display: inline-block;
```

Comment:
You might want to remove this line "display: inline-block;" if your intention was to center align the button with the rest of the text